### DISCUSSION TITLE
Message and reactions

### DISCUSSION THREAD

**Discussion by Art34222** [Rapptz/discord.py#9294] [Q&A]
How to get a message when a reaction is put on it?

**Comment by Art34222** [comment]
I need that when a certain reaction is pressed, a translated message is displayed, under which there was a reaction

**Comment by Art34222** [comment]
My function is already getting message id and translation language from react, but I can't figure out how to get message using message id

**Reply by Ombucha** [reply]
You can have a look at all of [these methods](https://discordpy.readthedocs.io/en/stable/search.html?q=fetch_message), and use one based on the data you have.
